package.json

```
@@ -50,7 +50,8 @@
   },
   "dependencies": {
     "@carbon/styles": "1.53.0",
-    "lit": "^3.0.0"
+    "lit": "^3.0.0",
+    "mermaid": "^11.2.0"
```

```suggestion
    "mermaid": "^11.2.0"
```
removing this as it's already in the `chat/package.json`